templates/cli/lib/commands/deploy.js.twig

```diff
@@ -227,14 +227,27 @@ const awaitPools = {
     },
 }
 
-const deploy = new Command("deploy")
-    .description(commandDescriptions['deploy'])
-    .configureHelp({
-        helpWidth: process.stdout.columns || 80
-    })
-    .action(actionRunner(async (_options, command) => {
-        command.help()
-    }));
+const deployResources = async ({ all, yes } = {}) => {
+    const actions = {
+        functions: deployFunction,
+        collections: deployCollection,
+        buckets: deployBucket,
+        teams: deployTeam,
+        messages: new Function()
```

Comment:
what exactly is this?

Reply:
The base branch has been updated to match the one that includes messaging implementation